octobot_evaluators/evaluator/abstract_evaluator.py

```diff
@@ -105,32 +115,55 @@ def get_is_time_frame_wildcard(cls) -> bool:
         """
         return True
 
-    def initialize(self):
-        set_tentacle_value(
-            matrix_id=self.matrix_id,
-            tentacle_type=self.get_eval_type(),
-            tentacle_value=None,
-            tentacle_path=get_matrix_default_value_path(
-                exchange_name=self.exchange_name,
-                tentacle_type=self.evaluator_type.value,
-                tentacle_name=self.get_name(),
-                cryptocurrency=self.cryptocurrency,
-                symbol=self.symbol,
-                time_frame=self.time_frame
-            )
-        )
+    def _get_tentacle_registration_topic(self, all_symbols_by_crypto_currencies, time_frames, real_time_time_frames):
```

Comment:
This method is only used for initialization right ?

Reply:
Yes, why ?

Reply:
I was trying to understand the purpose of this method, it's now clear :)